modules/collect/src/main/java/com/opengamma/strata/collect/timeseries/LocalDateDoubleTimeSeriesBuilder.java

```diff
@@ -156,9 +155,14 @@ public LocalDateDoubleTimeSeriesBuilder merge(LocalDate date, double value, Doub
    * @return this builder
    */
   public LocalDateDoubleTimeSeriesBuilder merge(LocalDateDoublePoint point, DoubleBinaryOperator operator) {
-    ArgChecker.notNull(point, "point");
```

Comment:
Did you not want to leave the `notNull` check here? 
The equivalent `put(LocalDateDoublePoint point)` has a similar check before delegating